.gitignore

```diff
@@ -12,4 +12,6 @@ logs
 results
 
 node_modules
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+
+.idea/
```

Comment:
Had no clue about this, thank you for sharing. Should I revert the gitignore to its last commit's state?